docs/How-to-Write-a-Beaker-Test-for-a-Module.md

```diff
@@ -31,12 +20,24 @@
   * beaker-rspec will use spec/acceptance/nodesets/default.yml node file
   * boxes will be re-used from previous run
   * boxes will be preserved post-testing
-* Nodes become corrupted with too many test runs/bad data and need to be refreshed then GOTO step 1
+* Nodes become corrupted with too many test runs/bad data and need to be refreshed then set `BEAKER_provision=yes`
 * Testing is complete and you want to clean up, run once more with `BEAKER_destroy` unset
   * you can also:
 
         cd .vagrant/beaker_vagrant_files/default.yml ; vagrant destroy --force
 
+###Supported ENV variables
```

Comment:
Since these are documented from the beaker-rspec [README](https://github.com/puppetlabs/beaker-rspec/blob/master/README.md), deleting them altogether in favor of a link to that seems like a better option here to me. This is so that when we update them, we won't have to update these docs in as many places. What do you think?